packages/client/src/views/UserAudit/UserAudit.tsx

```diff
@@ -295,15 +296,23 @@ export const UserAudit = () => {
         <Content
           title={user.name}
           icon={() => <UserAvatar name={user.name} avatar={user.avatar} />}
-          topActionButtons={[
-            <Status status={user.status || 'pending'} />,
-            <ToggleMenu
-              id={`sub-page-header-munu-button`}
-              toggleButton={<VerticalThreeDots />}
-              menuItems={getMenuItems(user.id as string, user.status as string)}
-              hide={(scope && !scope.includes('sysadmin')) || false}
-            />
-          ]}
+          topActionButtons={
+            userDetails?.systemRole === SystemRoleType.LocalSystemAdmin &&
+            userDetails?.primaryOffice?.id !== user?.primaryOffice?.id
+              ? [<Status status={user.status || 'pending'} />]
+              : [
+                  <Status status={user.status || 'pending'} />,
+                  <ToggleMenu
+                    id={`sub-page-header-munu-button`}
+                    toggleButton={<VerticalThreeDots />}
+                    menuItems={getMenuItems(
+                      user.id as string,
+                      user.status as string
+                    )}
+                    hide={(scope && !scope.includes('sysadmin')) || false}
```

Comment:
I wonder if these changes would be easier to understand if we'd add a new function to the module scope

```js
function canEditUserDetails(targetUser, loggedInUser, scopes) {
  if(!scope.includes('sysadmin')) {
    return false
  }
  if(loggedInUser.systemRole === SystemRoleType.NationalSystemAdmin) {
    return true
  }

  const usersInTheSameOffice = loggedInUser.primaryOffice?.id === targetUser?.primaryOffice?.id
  if(loggedInUser.systemRole === SystemRoleType.LocalSystemAdmin && usersInTheSameOffice) {
    return true  
  }

  return false
}

// ...
// ...
<Content
  title={user.name}
  icon={() => <UserAvatar name={user.name} avatar={user.avatar} />}
  topActionButtons={[
    <Status status={user.status || 'pending'} />,
    <ToggleMenu
      id={`sub-page-header-munu-button`}
      toggleButton={<VerticalThreeDots />}
      menuItems={getMenuItems(
        user.id as string,
        user.status as string
      )}
      hide={!canEditUserDetails(user, userDetails, scopes)}
```

Reply:
This is much easier to understand and maintain. Let's do it this way please